2020-02-09 02:42:23,289 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2020-02-09 02:44:01,107 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2020-02-09 02:45:23,216 - telegram.ext.dispatcher - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "prototype.py", line 86, in reply_to_messages
    combine_tags(message_json,current_document,all_tags)
  File "prototype.py", line 62, in combine_tags
    new_tags = list(set(current_document['tags'] + all_tags))
TypeError: 'NoneType' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ashoka/Documents/tattle_bot/venv/lib/python3.6/site-packages/telegram/ext/dispatcher.py", line 372, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/ashoka/Documents/tattle_bot/venv/lib/python3.6/site-packages/telegram/ext/handler.py", line 117, in handle_update
    return self.callback(update, context)
  File "prototype.py", line 129, in storing_data
    reply_to_messages(relevant_section,False)
  File "prototype.py", line 90, in reply_to_messages
    error_message()
TypeError: error_message() missing 1 required positional argument: 'message_json'
2020-02-09 02:50:27,142 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
2020-02-09 03:00:56,862 - telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
